src/test/java/com/java8/wordsplay/FileReaderTest.java

```diff
@@ -97,7 +97,7 @@ void shouldReturnWordWithSortedEntries() throws Exception {
                         )
                 )
                .entrySet()
-                .stream()
+               .stream()
                 . sorted( comparing(Entry::getKey)).collect(toList());
 
         wordMap.forEach( entry1 -> System.out.println(entry1.getKey() + " " + entry1.getValue()));
```

Comment:
- WordMap can be empty , then we would have an Optional<Map> coming out ,and we check that before we print ? - Could be a edge case but makes it complete.

Reply:
Good spot will fix